uniqueidentifier.go

```diff
@@ -0,0 +1,74 @@
+package mssql
+
+import (
+	"database/sql/driver"
+	"encoding/hex"
+	"errors"
+	"fmt"
+)
+
+type UniqueIdentifier [16]byte
```

Comment:
It seems more appropriate to me to keep the name congruent with the SQL Server data type that it represents.